adafruit_midi/note_off.py

```diff
@@ -36,17 +36,17 @@ class NoteOff(MIDIMessage):  # pylint: disable=duplicate-code
     LENGTH = 3
 
     def __init__(self, note, velocity=0, *, channel=None):
-        self._note = note_parser(note)
+        self.note = note_parser(note)
```

Comment:
You may want to add a `"""comment"""` after `self.note` so that it gets documented.

Reply:
Ah good idea. Will do.